MIGRATION.portable-stories.md

```diff
@@ -0,0 +1,368 @@
+        );
+      }
+
+      stories.forEach(({ name, story }) => {
+        // Instead of not running the test, you can create logic to skip it, flagging it accordingly in the test results.
+        const testFn = story.parameters.storyshots?.skip ? test.skip : test;
+
+        testFn(name, async () => {
+          const mounted = render(story());
+          // Ensures a consistent snapshot by waiting for the component to render by adding a delay of 1 ms before taking the snapshot.
+          await new Promise((resolve) => setTimeout(resolve, 1));
+          expect(mounted.container).toMatchSnapshot();
+        });
+      });
+    });
+  });
+});
+```
+
+Running this example will generate a single snapshot file (i.e., `__snapshots__/storybook.test.ts.snap`) with all the stories. However, if you were previously using a multi-snapshot configuration with the Storyshots addon, you can adjust the example above to include the [`jest-specific-snapshot`](https://github.com/igor-dv/jest-specific-snapshot) package. For example:
+
+```ts
+// storybook.test.ts
+
+// 👇 Augment expect with jest-specific-snapshot
+import 'jest-specific-snapshot';
+
+// ...Code omitted for brevity
+
+describe(options.suite, () => {
+  //👇 Add storyDir in the arguments list
+  getAllStoryFiles().forEach(({ filePath, storyFile, storyDir }) => {
+    // ...Previously existing code
+    describe(title, () => {
+      // ...Previously existing code
+      stories.forEach(({ name, story }) => {
+        // ...Previously existing code
+        testFn(name, async () => {
+          // ...Previously existing code
+
+          //👇 Define the path to save the snapshot to:
+          const snapshotPath = path.join(
+            storyDir,
+            options.snapshotsDirName,
+            `${componentName}${options.snapshotExtension}`
+          );
+          expect(mounted.container).toMatchSpecificSnapshot(snapshotPath);
+        });
+      });
+    });
+  });
+});
+```
+
+When the example above runs, it will generate individual snapshot files, one per story, using the following naming convention and location:
+
+```
+components/__snapshots__/Button.stories.storyshot
+components/__snapshots__/Header.stories.storyshot
+components/__snapshots__/Page.stories.storyshot
+```
+
+### 3 - Remove Storyshots from your project
+
+After you confirm that the portable stories solution suits your needs, delete your old storyshots test file and uninstall `@storybook/addon-storyshots` from your project.
+
+### 4 - (Optional) Extend your testing coverage
+
+The examples above will give you the closest possible experience with the Storyshots addon. However, if you are using Storyshots for other use cases, such as accessibility testing, image snapshot testing, or different testing scenarios, you can extend them to suit your needs or extend your testing solution to use the [Storybook test-runner](https://github.com/storybookjs/test-runner), that provides out-of-the-box solutions for such use cases.
```

Comment:
> that provides out-of-the-box solutions for such use cases.
That's not true for some cases, such as accessibility and image snapshot testing. People still need to install dependencies and follow recipes